mkosi/__init__.py

```diff
@@ -2699,22 +2710,30 @@ def install_arch(args: MkosiArgs, root: Path, do_run_build_script: bool) -> None
                 Architecture = auto
                 Color
                 CheckSpace
-                SigLevel = Required DatabaseOptional TrustAll
+                SigLevel = {sig_level}
                 ParallelDownloads = 5
 
+                {f"Include = {args.repos_dir}/*" if args.repos_dir else ""}
```

Comment:
Moving the include up here changes the precedence of repositories defined in there to be higher than core, I'm not sure that's what we want or at least it's an unrelated change.

Reply:
ah got it - I didn't know the order mattered, fixed